balancer-js/src/pool-stable/calculate-balance-given-invariant.ts

```diff
@@ -0,0 +1,87 @@
+import { ONE, SolidityMaths } from '@/lib/utils/solidityMaths';
+import { AMP_PRECISION } from '@/lib/utils';
+import { BalancerError, BalancerErrorCode } from '@/balancerErrors';
+
+export const calculateBalanceGivenInvariantAndAllOtherBalances = ({
```

Comment:
What is the difference between this and what is already implemented in the [SOR](https://github.com/balancer-labs/balancer-sor/blob/master/src/pools/stablePool/stableMathBigInt.ts#L433)? 🤔

Reply:
Update: The method is implemented on SOR, @brunoguerios passed me the name, is `_getTokenBalanceGivenInvariantAndAllOtherBalances`